templates/tasks/edit_task.html

```diff
@@ -27,4 +27,53 @@
     </div>
     <button type="submit" class="text-white bg-blue-700 hover:bg-blue-800 focus:ring-4 focus:outline-none focus:ring-blue-300 font-medium rounded-lg text-sm w-full sm:w-auto px-5 py-2.5 text-center dark:bg-blue-600 dark:hover:bg-blue-700 dark:focus:ring-blue-800">Submit</button>
 </form>
+
+<script>
```

Comment:
[nit] Since this script is getting quite a lot, I think it would be best if this was put in its JS file, for better maintainability

Reply:
moved it to its own separate file on next update